NuRadioReco/utilities/fft.py

```diff
@@ -30,23 +30,46 @@
 
 import numpy as np
 
+def freqs(num_samples, sampling_rate):
+    """
```

Comment:
I'm trying to think if there's a clean way to extend this such that a user can call this with the number of frequencies instead of `num_samples`, by adding an optional switch or having both `num_samples` and `num_freqs` or something similar. But I can't think of one that I like sufficiently, and I don't know that we _need_ this.